fastchat/model/model_adapter.py

```diff
@@ -1256,6 +1256,20 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("starchat")
 
 
+class MistralAdapter(BaseModelAdapter):
+    """The model adapter for Mistral models"""
+
+    def match(self, model_path: str):
+        mp = model_path.lower()
+        return "mistral" in mp
+
+    def load_model(self, model_path: str, from_pretrained_kwargs: dict):
+        raise NotImplementedError()
+
+    def get_default_conv_template(self, model_path: str) -> Conversation:
+        return get_conv_template("mistral")
```

Comment:
If it is the same as Llama2, we can just return "llama-2"

Reply:
correct but there are some differences